Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm

```diff
@@ -152,11 +152,13 @@ static float toNSURLSessionTaskPriority(WebCore::ResourceLoadPriority priority)
 
 #if USE(CFNETWORK_CONTENT_ENCODING_SNIFFING_OVERRIDE)
     if (contentEncodingSniffingPolicy == WebCore::ContentEncodingSniffingPolicy::Disable)
-        [mutableRequest _setProperty:@YES forKey:bridge_cast(kCFURLRequestContentDecoderSkipURLCheck)];
+        // FIXME: This is a static analyzer false-positive due to the @YES/@NO constants.
```

Comment:
I think because of this comment the if statement needs braces, no?

Also, it would be nice if there was an associated bug this comment and comments like it could point to so we know when to clean it up.

Reply:
Though that bug should be in llvm, right?

I'm not sure about the braces being needed. Is that for style reasons or? AFAIK they are not strictly required (and the style checker seems ok with it).